repos/system_upgrade/common/actors/peseventsscanner/libraries/pes_events_scanner.py

```diff
@@ -439,8 +439,14 @@ def replace_pesids_with_repoids_in_packages(packages, source_pkgs_repoids):
             message='packages may not be installed or upgraded due to repositories unknown to leapp:',
             skipped_pkgs=packages_without_known_repoid,
             remediation=(
-                'Please file a bug in http://bugzilla.redhat.com/ for leapp-repository component of '
-                'the Red Hat Enterprise Linux product.'
+                'In case the listed repositories are mirrors of official repositories for RHEL'
+                ' (provided by Red Hat on CDN)'
+                ' and their repositories IDs has been customized you can change'
+                ' the configuration to use the official IDs instead to fix the problem.'
```

Comment:
```suggestion
                ' and their repositories IDs has been customized, you can change'
                ' the configuration to use the official IDs instead of fixing the problem.'
```